pages/[id]/intro.js

```diff
@@ -0,0 +1,247 @@
+import styled from 'styled-components';
+import { useState, useEffect } from 'react';
+import Page from '../../components/page';
+import Card, {
+  CardHeader,
+  CardTitle,
+  CardContent,
+  CardButtonContainer,
+} from '../../components/card';
+import { COLOR, EDIT } from '../../constants';
+import Button from '../../components/button';
+import TextBox from '../../components/textbox';
+import {
+  updateHackathonField,
+  getHackathon,
+  getHackathonPaths,
+  getHackathons,
+} from '../../utility/firebase';
+
+const HeaderText = styled.p`
+  border: none;
+  overflow: hidden;
+  background-color: ${COLOR.BACKGROUND};
+  outline: none;
+  font-size: 24px;
+  padding-bottom: 13px;
+  color: ${COLOR.TEXT};
+  font-weight: 600;
+`;
+
+const ContentText = styled.p`
+  border: none;
+  overflow: hidden;
+  background-color: ${COLOR.BACKGROUND};
+  outline: none;
+  color: ${COLOR.TEXT};
+`;
+
+const StyledCancel = styled.button`
+  font-size: 16px;
+  cursor: pointer;
+  border-bottom: 2px solid ${COLOR.BLACK};
+  margin-left: 675px;
+  margin-right: 40px;
+  border: none;
+  outline: none;
+  background-color: ${COLOR.BACKGROUND};
+`;
+
+const StyledHeader = styled.p`
```

Comment:
You can probably just call this `Header`. I've mostly seen `Styled` being used when you modify an existing component. For example, if you wanted to use the existing `Button` component but you needed it to be larger in this page, you would call it `StyledButton`

Reply:
And maybe its better to call this a label instead of a header? I like to reserve "Header" for actual header elements ie `h1` `h2`